mlir/utils/jenkins/Jenkinsfile

```diff
@@ -256,47 +256,143 @@ pipeline {
                             }
                         }
                         stage("Build MIOpen with HIP") {
+                           steps {
+                               dir('MIOpen') {
+                                   getAndBuildMIOpen('--prefix ./MIOpenDeps',
+                                                     '''-DMIOPEN_BACKEND=HIP
+                                   -DCMAKE_PREFIX_PATH="${WORKSPACE}/MIOpen/MIOpenDeps"
+                                   -DCMAKE_INSTALL_PREFIX=${WORKSPACE}/MIOpen/build/MIOpenInstallDir
+                                   ''')
+                                   sh 'cd build; make -j $(nproc)'
+                                   sh 'cd build; make install'
+                               }
+                                stash name:"MIOpen-kernels",\
+                                   includes: "MIOpen/build/**"
+                           }
```

Comment:
Please correct me if I'm wrong but I believe you need to clean up the cmake build cache (after stash) otherwise it will get used by next stage.

Reply:
You're right. The build directory is deleted in getAndBuildMIOpen(), so we don't have to worry about it here.

Reply:
Ok gotcha